Makefile

```diff
@@ -18,14 +18,14 @@ deps:
 	@echo '-> Downloading Hugo CLI to ./deps '; \
 	mkdir -p deps/; \
 	if [[ "$(distname)" == "Windows" ]]; then \
-	  wget --no-check-certificate -O hugo.zip $(url).zip; \
+	  curl -k $(url).zip -o hugo.zip; \
 	  unzip hugo.zip -d deps; \
 	else \
-	  wget -O hugo.tar.gz $(url).tar.gz; \
+	  echo $(url).tar.gz; \
```

Comment:
Why is the `echo` in here?

Reply:
Good question. it was probably for debugging. I will remove it.